portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -57,4 +60,28 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(gson.toJson(comments));
   }
+
+  /**
+   * Convert a string into a integer. If not a natural number,
```

Comment:
nit: "an"

Reply:
Good catch! I was playing around with the javadoc wording and I missed this one